.github/workflows/publish-beta-release.yml

```diff
@@ -0,0 +1,73 @@
+name: Publish Beta Release to Github Only
+
+on: 
+  workflow_dispatch:
+
+jobs:
+  save_artifacts:
+    name: 'Build VSIX Files'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '16.13.x'
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.ref }}
+      - run: npm install -g lerna
+      - run: npm install
+      - run: npm run compile
+      - run: npm run vscode:package
+      - name: Stage Artifacts
+        run: |
+          mkdir extensions
+          find packages -name "*.vsix" -type f -exec cp {} ./extensions \;
+          cp ./packages/salesforcedx-vscode/CHANGELOG.md ./extensions
```

Comment:
thought for the future. it's nice when all the steps are just running an npm script.  Might consider adding new script related to actions

Reply:
Like a JS script, or a separate actions file, like there is for the ghConfig? As an action, this could be something like 'Save Extensions'